WeeklyTask/WeeklyTaskService.cs

```diff
@@ -6,103 +6,160 @@
 
 namespace WeeklyTask
 {
+    delegate void GetMessage(string message);
+    delegate void WriteOutput(string text);
+    delegate string ReadInput();
+    
     internal class WeeklyTaskService
     {
+        GetMessage _mes= GetMessageAboutUpdateTask;
+        WriteOutput _writeOutput;
+        ReadInput _readInput=()=>Console.ReadLine();
```

Comment:
Use another method SetReader similar to `RegisterExchanger`, do not reference console here